code/modules/mob/living/silicon/robot/sprites/medical.dm

```diff
@@ -251,7 +251,7 @@
 		SP.attack_verb = list("batted", "pawed", "bopped", "whapped")
 
 /datum/robot_sprite/dogborg/tall/surgical/raptor
-	name = "Raptor V-4"
+	name = "Raptor Surgical"
```

Comment:
Im unsure about changing the names considering the modules are separate and there is no overlap, and having same name for same 'sprite type' is good for recognizability. Same applies to Crisis and Syndie modules below.

Reply:
You are right about the station medical ones. Can just revert though. I forgot they were separate on virgo specific. Chomp has them as one. But for the syndicate ones, I could not find a means to separate the selection of the three syndi models. As in, if you were a protector borg, the medi and engi raptor would appear

Reply:
I think I found the cause of that. You have /datum/robot_sprite/dogborg/tall/syndie defined 3 different times with different sprites. You might want to replace /syndie with actual individual module names, like /datum/robot_sprite/dogborg/tall/protector, /datum/robot_sprite/dogborg/tall/mechanist and /datum/robot_sprite/dogborg/tall/combat_medic

Reply:
Realizing I messed up and am behind on a unGODLY amount of commits. I am just going to close this one and open a fixed on.